app/assets/stylesheets/unstructured/_pages.scss

```diff
@@ -121,6 +121,8 @@ a.btn-icon:hover { color: $dark-grey; }
 .pages.crowd,
 .pages.wallets,
 .woc.index,
+.start_group.new,
+body.passwords,
 .we_the_people.index {
```

Comment:
Yeah it's real bad, eh?

I was happy to get all of the non-angular pages into one application.html.haml template at least.